Junit/src/IPL.java

```diff
@@ -0,0 +1,364 @@
+            return -1;
+        }
+        return 0;
+    }
+}
+
+class RunComparator implements Comparator<Player>{
+    public int compare(Player p1,Player p2){
+        if(p1.getRuns() < p2.getRuns()){
+            return 1;
+        }
+        else if(p1.getRuns() > p2.getRuns()){
+            return -1;
+        }
+        return 0;
+    }
+}
+
+
+class RunWicketComparator implements Comparator<Player>{
+    public int compare(Player p1,Player p2){
+        if(p1.cumulativeScore() < p2.cumulativeScore()){
+            return 1;
+        }
+        else if(p1.cumulativeScore() > p2.cumulativeScore()){
+            return -1;
+        }
+        return 0;
+    }
+}
+
+
+
+
+class IPL{
+
+    // for putting all players into list of player objects.
+    public static List<Player> putPlayers(){
+
+        List<Player> listOfPlayers = new ArrayList<Player>();
+
+        try{
+
+            BufferedReader reader = new BufferedReader(new FileReader("./src/IPL_2021_data.csv"));
+
+            // for passing the header
+            reader.readLine();
+            String line = "";
+            while((line = reader.readLine()) != null){
+                String[] playerArray = line.split(",");
+                listOfPlayers.add(new Player(playerArray[0],playerArray[1],playerArray[2],Integer.valueOf(playerArray[3]),Integer.valueOf(playerArray[4]),Double.valueOf(playerArray[5]),Double.valueOf(playerArray[6]),Integer.valueOf(playerArray[7])));
+            }
+
+            reader.close();
+        }
+        catch(Exception e){
+            System.out.println("File not found "+ e);
+        }
+
+        return listOfPlayers;
+    }
+
+    // for fixing the match between team and stadium and storing in a csv file
+    public static void fixMatch(){
+        final int totalTeams = 8;
+        String[] listOfTeams = new String[totalTeams];
+        String[] homeTeamPlace = new String[totalTeams];
+
+        listOfTeams[0] =  "CSK";
+        listOfTeams[1] =  "PBKS";
+        listOfTeams[2] =  "RCB";
+        listOfTeams[3] =  "SRH";
+        listOfTeams[4] =  "MI";
+        listOfTeams[5] =  "RR";
+        listOfTeams[6] =  "DC";
+        listOfTeams[7] =  "KKR";
+
+        homeTeamPlace[0] = "HOME CSK";
+        homeTeamPlace[1] = "HOME PBKS";
+        homeTeamPlace[2] = "HOME RCB";
+        homeTeamPlace[3] = "HOME SRH";
+        homeTeamPlace[4] = "HOME MI";
+        homeTeamPlace[5] = "HOME RR";
+        homeTeamPlace[6] = "HOME DC";
+        homeTeamPlace[7] = "HOME KKR";
+
+
+        ArrayList<String> matchList = new ArrayList<String>();
+        for(int i=0;i<totalTeams;i++){
+            for(int j=i+1;j<totalTeams;j++){
+                matchList.add(listOfTeams[i] + "," + listOfTeams[j] + "," + homeTeamPlace[i] + "\n");
+                matchList.add(listOfTeams[i] + "," + listOfTeams[j] + "," + homeTeamPlace[j] + "\n");
+            }
+        }
+
+        Collections.shuffle(matchList);
+
+
+
+        try{
```

Comment:
segregate methods into meaningful blocks, where, each method is responsible for only one operation. in this case, this operation of writing it to a file should have been offloaded to a separate utility method.